--- python/pkgsrcrecords.old.cc	2015-05-07 19:21:41.000000000 +0300
+++ python/pkgsrcrecords.cc	2014-04-25 15:00:16.000000000 +0400
@@ -75,34 +75,10 @@
    return HandleErrors(Py_None);
 }
 
-static char *doc_PkgSrcRecordsStep =
-    "step() -> bool\n\n"
-    "Go to the source package. Each call moves\n"
-    "the position of the records parser forward. If there are no\n"
-    "more records, return None. If the lookup failed this way,\n"
-    "access to any of the attributes will result in an AttributeError.";
-static PyObject *PkgSrcRecordsStep(PyObject *Self,PyObject *Args)
-{
-   PkgSrcRecordsStruct &Struct = GetCpp<PkgSrcRecordsStruct>(Self);
-
-   if (PyArg_ParseTuple(Args,"") == 0)
-      return 0;
-
-   Struct.Last = (pkgSrcRecords::Parser*)Struct.Records->Step();
-   if (Struct.Last == 0) {
-      Struct.Records->Restart();
-      Py_INCREF(Py_None);
-      return HandleErrors(Py_None);
-   }
-
-   return PyBool_FromLong(1);
-}
-
 static PyMethodDef PkgSrcRecordsMethods[] =
 {
    {"lookup",PkgSrcRecordsLookup,METH_VARARGS,doc_PkgSrcRecordsLookup},
    {"restart",PkgSrcRecordsRestart,METH_VARARGS,doc_PkgSrcRecordsRestart},
-   {"step",PkgSrcRecordsStep,METH_VARARGS,doc_PkgSrcRecordsStep},
    {}
 };
 
